src/harness/request_handler.py

```diff
@@ -132,34 +143,32 @@ def _Request(url, request, config, is_post_method):
   else:
     logging.info('GET Request to URL %s', url)
 
-  error_occurred = False
-
+  error = None
   for attempt_count in range(MAX_REQUEST_ATTEMPT_COUNT):
     try:
       conn.perform()
-      error_occurred = False
       break
     except pycurl.error as e:
       # e contains a tuple (libcurl_error_code, string_description).
       # See https://curl.haxx.se/libcurl/c/libcurl-errors.html
-      error_occurred = True
+      error = e
       logging.warning(str(CurlError(e.args[1], e.args[0])))
       time.sleep(REQUEST_ATTEMPT_DELAY_SECOND)
     except Exception as e:
-      error_occurred = True
+      error = e
       logging.warning(str(e))
       time.sleep(REQUEST_ATTEMPT_DELAY_SECOND)
 
-  if error_occurred:
+  if error:
     logging.error('Connection to Host Failed after %d attempts' %MAX_REQUEST_ATTEMPT_COUNT)
-    raise
+    raise error
 
   http_code = conn.getinfo(pycurl.HTTP_CODE)
   conn.close()
-  body = response.getvalue()
+  body = response.getvalue().decode('utf-8')
```

Comment:
That is strange for this specific piece of code, since in fact you will see there is no functional change: the .decode('utf-8') has just been moved from the return at the end to here.

Could you retry running now that all typo are fixed ?